concept-anchor.md

```diff
@@ -0,0 +1,36 @@
+## Title
+Concept Anchor for discussions with Management
+
+## Patlet
+Those wanting to bring InnerSource to organizations have difficulty starting a discussion with management. This can sometimes be made easier by anchoring InnerSource to more well known concepts such as Agile / DevOps / Lean.
+
+## Problem
+Often InnerSource is a term that's novel to organizations, making it often difficult to start a conversation with management about adopting InnerSource.
+
+## Context
+An individual sees an opportunity for InnerSource to make an impact within an organization. They would like to discuss InnerSource with management. Being a relatively traditional and bureaucratic company, it is difficult to broach the subject of InnerSource with management as they are reluctant to engage on a topic that they've neither heard of nor know anything about.
+
+These organizations might have an existing strategy, such as Agile, Lean, or DevOps transformation. To gain mindshare, it might be useful to start with these existing, well known concepts and build InnerSource on top of these concepts.
```

Comment:
Can you elaborate on what you mean by `mindshare`?

Reply:
@mishari Do you think IS would help solving some of the challenges in those existing strategy therefore getting management's attention to lay out how IS would work with those existing strategy?

Reply:
> Can you elaborate on what you mean by `mindshare`?

Webster defines mindshare as "a controlling or predominant hold of one's attention that is gained especially by marketing ploys" the concept is basically correct.

Reply:
> @mishari Do you think IS would help solving some of the challenges in those existing strategy therefore getting management's attention to lay out how IS would work with those existing strategy?

I do.

With Agile, if A depends on B, but B's priority is not A, IS allows A to put the work on their backlog, win win for A, B, and all the customers. It's also a win for Agile because, Agile is unstuck :-P

With DevOps, having A submit production ready code to B efficiently will also mean an improvement in tooling for automated tests, rollback capabilities, the ability to accept and manage greater change and innovation with less risks, in line with DevOps principles. Change is only scary if you don't change very often.

Reply:
I am not sure if "mindshare" will be easy enough for readers.
However given that this is a Donut in level 1, I would leave this as is and improve the pattern when we level it up later.